lib/ProductOpener/Display.pm

```diff
@@ -1201,8 +1190,6 @@ sub display_text ($request_ref) {
 		$html .= search_and_display_products($request_ref, {}, "last_modified_t_complete_first", undef, undef);
 	}
 
-	$html =~ s/\[\[query:(.*?)\]\]/$replace_query->($1)/eg;
```

Comment:
This feature (including the result of a query inside a text) is not used anymore, so I think it's ok to remove it indeed.

```
lang/fr/texts/index.old.html:[[query:{"lc":"fr","sort_by":"last_modified_t_complete_first"}]]
lang/fr/texts/index-test.html:[[query:{"lc":"fr","sort_by":"last_modified_t"}]]

```